### PR TITLE
Fix Quais-5 tests

### PR DESCRIPTION
**Pull request by DenisIvanov26**

### CODE REVIEW

File: packages/abstract-provider/src.ts/index.ts [hunk 1]
```diff
@@ -82,14 +81,27 @@ export interface _Block {
...
+    extTransactionsRoot: string;
     stateRoot: string;
+    location: string;
+    manifestHash: Array<String>;
+    mixHash: string;
     transactionsRoot: string;
     receiptsRoot: string;
+    order: number;
+    parentDeltaS: Array<String>;
+    sha3Uncles: String;
```
Comment (wizeguyy) [inline packages/abstract-provider/src.ts/index.ts:94]: why are some types `String` and others `string`? Does capitalization matter? Are they different types?

File: packages/providers/src.ts/formatter.ts [hunk 2]
```diff
@@ -46,24 +46,23 @@ export class Formatter {
         const data = this.data.bind(this);
         const hash = this.hash.bind(this);
         const hashArray = this.hashArray.bind(this);
+        const hashArrayAnyLength = this.hashArrayAnyLength.bind(this);
         const hex = this.hex.bind(this);
         const number = this.number.bind(this);
-        const type = this.type.bind(this);
+        const etxs = this.etxs.bind(this)
```
Comment (wizeguyy) [inline packages/providers/src.ts/formatter.ts:53]: missing semicolon?

File: packages/providers/src.ts/formatter.ts [hunk 3]
```diff
@@ -72,23 +71,27 @@ export class Formatter {
             maxPriorityFeePerGas: Formatter.allowNull(bigNumber),
             maxFeePerGas: Formatter.allowNull(bigNumber),
 
-            gasLimit: bigNumber,
+            //gasLimit: bigNumber,
```
Comment (wizeguyy) [inline packages/providers/src.ts/formatter.ts:75]: just delete it if you don't need it. we can always dig through git history if you want to find it again

File: packages/providers/src.ts/formatter.ts [hunk 4]
```diff
@@ -72,23 +71,27 @@ export class Formatter {
...
-            creates: Formatter.allowNull(address, null),
+            s: Formatter.allowNull(hex),
+            v: Formatter.allowNull(hex),
 
             raw: Formatter.allowNull(data),
 
             gas: Formatter.allowNull(bigNumber),
-            input: Formatter.allowNull(data),
-            sender: Formatter.allowNull(address),
+            //sender: Formatter.allowNull(address),
```
Comment (wizeguyy) [inline packages/providers/src.ts/formatter.ts:88]: same as above... delete the cruft

File: packages/providers/src.ts/formatter.ts [hunk 5]
```diff
@@ -177,7 +200,7 @@ export class Formatter {
             blockHash: Formatter.allowNull(hash),
             transactionIndex: number,
 
-            removed: Formatter.allowNull(this.boolean.bind(this)),
+            //removed: Formatter.allowNull(this.boolean.bind(this)),
```
Comment (wizeguyy) [inline packages/providers/src.ts/formatter.ts:204]: :eyes:

File: packages/abstract-provider/src.ts/index.ts (hunk 1 above)
Comment (DenisIvanov26) [reply packages/abstract-provider/src.ts/index.ts]: Capitalization does matter as one is a primitive type while the other is a string object wrapper. Changed all to string.

**Review by wizeguyy — APPROVED** [review]

**Review by wizeguyy — CHANGES_REQUESTED** [review]
You added two more commits, which are just tweaks of work done in prior commits. Please squash them into the appropriate commits. Otherwise, good to go

**Comment by DenisIvanov26** [comment]
> You added two more commits, which are just tweaks of work done in prior commits. Please squash them into the appropriate commits. Otherwise, good to go

Resolved

**Review by jdowning100 — APPROVED** [review]

**Review by wizeguyy — APPROVED** [review]

**Pull request merged**